Tasks/app-store-release/app-store-release.ts

```diff
@@ -85,6 +128,15 @@ async function run() {
                 credentials.appSpecificPassword = tl.getInput('appSpecificPassword', true);
                 credentials.fastlaneSession = tl.getInput('fastlaneSession', false);
             }
+        } else if (authType === 'ApiKey') {
+            isUsingApiKey = true;
+            apiKey = {
+                key_id: tl.getInput('apiKeyId', true),
+                issuer_id: tl.getInput('apiKeyIssuerId', true),
+                key: tl.getInput('apiKeyContent', true),
+                in_house: tl.getBoolInput('apiKeyInHouse', false),
+                is_key_content_base64: true
```

Comment:
Hi Peter, thanks for the feedback. I did set `is_key_content_base64` on purpose because otherwise the content of the key string was being corrupted. See this commit: https://github.com/microsoft/app-store-vsts-extension/pull/189/commits/a98d239d12d4159cd38bd5322990ee010f2c4b95

Reply:
Strange, I had no problems pasting the multiline content, but I used a service connection. So may be the problem is when you enter it in the build definition. Ok, I guess this should be fine then (sorry, I missed that this needs to be base64 of the whole P8 file, I thought it was just the base64 that is inside the file and it was failing for me when I used it like this)

Reply:
No problem :)